src/detectors/PHP/PHPLanguageDetector.ts

```diff
@@ -0,0 +1,30 @@
+import { ILanguageDetector } from '../ILanguageDetector';
+import { injectable, inject } from 'inversify';
+import { LanguageAtPath, ProgrammingLanguage } from '../../model';
+import { IFileInspector } from '../../inspectors/IFileInspector';
+import { Types } from '../../types';
+import { fileExtensionRegExp, sharedSubpath } from '../utils';
+import { uniq } from 'lodash';
+import * as nodePath from 'path';
+
+@injectable()
+export class PHPLanguageDetector implements ILanguageDetector {
+  private fileInspector: IFileInspector;
+  constructor(@inject(Types.IFileInspector) fileInspector: IFileInspector) {
+    this.fileInspector = fileInspector;
+  }
+
+  async detectLanguage(): Promise<LanguageAtPath[]> {
+    const result: LanguageAtPath[] = [];
+
+    const phpFiles = await this.fileInspector.scanFor(fileExtensionRegExp(['php']), '/');
```

Comment:
@Sehsyha What about to determine PHP language with `composer.phar` file? Is it common usage with PHP Composer etc.?

Reply:
I didn't remember which file I could find to determine if the language was PHP or not :/ I add it to this PR immediately ;)